lib/modules/manager/gitlabci-include/extract.spec.ts

```diff
@@ -29,6 +31,28 @@ describe('modules/manager/gitlabci-include/extract', () => {
       expect(res?.deps).toHaveLength(3);
     });
 
+    it('extracts multiple embedded include blocks', () => {
+      const res = extractPackageFile(yamlWithTriggerRef);
+      expect(res?.deps).toHaveLength(2);
+      expect(res?.deps).toMatchSnapshot([
```

Comment:
use toMatchObject instead 
```suggestion
      expect(res?.deps).toMatchObject([
```